index.js

```diff
@@ -0,0 +1,22 @@
+const formElement = document.querySelector("form");
+
+formElement.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  const heightInput = formElement.height.value;
+  const weightInput = formElement.weight.value;
+  let bmi = weightInput / heightInput ** 2;
+  const text = document.querySelector("p");
+
+  text.textContent = `Your body mass index is + ${bmi.toFixed(2)}`;
+
+  if (bmi < 18.5 && bmi >= 30) {
```

Comment:
I think this should be an OR `||` Operator in between the values, right?

Reply:
Yes, that`s right. Tks!